toolbarview.py

```
@@ -99,8 +109,11 @@ def on_rc_rx(self, value):
         self.rcRxStatus.color = self.rxOnColor if value else self.rxOffColor
         self._rcRxDecay()        
 
-    def on_tele_status(self, connected):
+    def on_tele_status(self, status):
         if not self.teleStatus:
             self.teleStatus = kvFind(self, 'rcid', 'teleStatus')
-        self.teleStatus.color = self.txOnColor if connected else self.txOffColor
+        try:        
+            self.teleStatus.color = self.telemetry_color[status]
+        except:
+            pass
```

We should at least log something here if there's an error, I've torn my hair out debugging things in this code because errors were silently swallowed.